_includes/head.html

```diff
@@ -0,0 +1,17 @@
+  <head>
+    <meta charset="utf-8">
+    
+    <title>{% if page.title %} {{ page.title }} | {% endif %} {{ site.name }}</title>
+    <meta name="description" content="{{site.description}}">
+
+    <link rel="stylesheet" href="https://img1.wsimg.com/ux/1.3.50-brand/css/uxcore.min.css" media="all">
+    <link rel="stylesheet" href="https://img1.wsimg.com/wrhs/d395a75515b2ebef0ec1d145e6d9ee97/appheader.min.css" media="all">
+
+    <style>
+      body {
+        background: #e8e8e8;
+      }
+    </style>
+  </head>
+
```

Comment:
we only need one empty line, right?

Reply:
Oops, I forgot to set EOL characters in my IDE and after correcting that added too many here. :)